README.md

```diff
@@ -96,6 +96,20 @@ Validates presence/absence of a value.
 }
 ```
 
+It can also accept an `options` object, containing a `presence` and an optional `message`:
```

Comment:
I think we can remove this bit, and mention on L87 that all validators take a custom message option. You could link to the `custom-validation-messages` heading